### PR TITLE
Performance: stop using pkg_resources.declare_namespace('azure')

### PR DESCRIPTION
**Pull request by johanste**
Importing pkg_resources slowed down startup performance significantly - especially on slow/cpu starved machines. This change takes advantage of the azure management libraries update to not loading the pkg_resource module and follows suite in how it lays down the modules. 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [N/A] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [N/A] Each command and parameter has a meaningful description.
- [N/A] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/azure-cli-core/azure/cli/core/__init__.py [hunk 1]
```diff
@@ -2,7 +2,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
```
Comment (lmazuel) [inline src/azure-cli-core/azure/cli/core/__init__.py]: Is the package of this file a meta-package?

File: src/azure-cli-nspkg/setup.py [hunk 2]
```diff
@@ -40,6 +40,9 @@
     url='https://github.com/Azure/azure-cli',
     zip_safe=False,
     classifiers=CLASSIFIERS,
+    install_requires=[
+        'azure-nspkg>=2.0.0'
```
Comment (lmazuel) [inline src/azure-cli-nspkg/setup.py]: This is not supposed to be required

File: src/azure-cli-testsdk/setup.cfg [hunk 3]
```diff
@@ -1,2 +1,3 @@
 [bdist_wheel]
 universal=1
+init-file-root=azure-cli-testsdk
```
Comment (lmazuel) [inline src/azure-cli-testsdk/setup.cfg]: Why rename that?

File: src/command_modules/azure-cli-dla/setup.py [hunk 4]
```diff
@@ -48,13 +55,12 @@
     author_email='azpycli@microsoft.com',
     url='https://github.com/Azure/azure-cli',
     classifiers=CLASSIFIERS,
-    namespace_packages=[
+    packages=[
```
Comment (lmazuel) [inline src/command_modules/azure-cli-dla/setup.py]: I don't get this change :(

File: src/azure-cli-nspkg/setup.py (hunk 2 above)
Comment (johanste) [inline src/azure-cli-nspkg/setup.py]: The version range or the azure-nspkg entry alltogether?

Comment (lmazuel) [inline src/azure-cli-nspkg/setup.py]: hum ok, I didn't realize this one was a nspkg as well!

**Comment by derekbekoe** [comment]
I've taken a look and overall it looks good to me.
Does this mean that https://github.com/derekbekoe/azure-multiapi-storage-python/ also has to include the bdist_wheel extension?

**Comment by derekbekoe** [comment]
Also, do we need a CI script to verify that the command modules have the `azure_bdist_wheel.py` files present?

**Pull request closed without merging**